app/policies/two_factor_authentication/webauthn_policy.rb

```diff
@@ -30,7 +30,7 @@ def roaming_enabled?
     end
 
     def visible?
-      true
+      platform_configured? || IdentityConfig.store.platform_auth_set_up_enabled
     end
```

Comment:
This class is used both for roaming and platform authenticators. In the places we're checking `visible?`, should we parameterize the type of authenticator we're checking?

Reply:
Or we could create separate methods `platform_visible?` and `roaming_visible?`, which would at least be consistent with how there's `platform_configured?` and `roaming_configured?`.